scripts/process_data_spine-generic.sh

```diff
@@ -36,16 +36,20 @@ echo "PATH_RESULTS: ${PATH_RESULTS}"
 echo "PATH_LOG: ${PATH_LOG}"
 echo "PATH_QC: ${PATH_QC}"
 
+# Save script path
+PATH_DERIVATIVES="${PATH_DATA}/derivatives/labels"
+echo $PATH_DERIVATIVES
```

Comment:
just a nitpick to be consistent with `echo` commands above

```suggestion
echo "PATH_DERIVATIVES: ${PATH_DERIVATIVES}"
```